extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -284,6 +284,9 @@ export class AmpAdXOriginIframeHandler {
    * @private
    */
   handleOneTimeRequest_(requestType, getter) {
+    if (!this.iframe) {
```

Comment:
intended?

Reply:
Yeah, otherwise throws a type error.

Reply:
seems risky, what if this.iframe is null?

Reply:
Then `listenFor` would not work :)